pypff-stubs/pypff/__init__.pyi

```diff
@@ -0,0 +1,18 @@
+"""
+The stubs file for pypff, a python wrapper for libpff
+
+We base our understanding of the structure of each class through reading the
+class definitions in [1]. One peculiarity of the pypff library are the large amount
+of unnecessary "getter" methods that fetch class instance variables. 
+
+[1] https://github.com/Everlaw/libpff/tree/master/pypff
+"""
+
+from .File import file as file
```

Comment:
love the idea of dividing this several modules! any reason we're using pascal case for the files, when both python convention, and the original source files, use snake case?

Reply:
What do you mean by pascal case here?

`File.pyi` is capitalized per python convention

whereas, `file` is lowercase due to how its defined in [1]. [2] for documentation reference

[1] https://github.com/libyal/libpff/blob/main/pypff/pypff.c#L636
[2] https://docs.python.org/3/c-api/module.html#c.PyModule_AddObject

Reply:
9d129e00308d1f911d7dec320e94bc949626d171 :)